lib/src/Base/Geom/openturns/Mesh.hxx

```diff
@@ -52,7 +52,8 @@ public:
 
   /** Parameters constructor */
   Mesh(const Sample & vertices,
-       const IndicesCollection & simplices);
+       const IndicesCollection & simplices,
+       const Bool checkMeshValidity = ResourceMap::GetAsBool("Mesh-CheckValidity"));
```

Comment:
I suggest to leave the attribute out of the ctor and keep separate accessors,
(we have ctors with several bools and its confusing)

Reply:
ah

Reply:
what do you suggest? Having a key but without introducing it in ctor?

Reply:
yes, an attribute that would have an accessor, or just using the key

Reply:
Couldn't we do things this way:

- Always use ``ResourceMap::GetAsBool("Mesh-CheckValidity"))`` (so we don't provide the option to change the behavior in the ctor, which is @jschueller's concern).
- Set the default "Mesh-CheckValidity" to `True`.
- If a user thinks mesh checking takes too long, that user can simply set the ResourceMap key  "Mesh-CheckValidity" to `False`. Then it becomes the user's responsibility to call `isValid` whenever appropriate.

Reply:
Finally I think I will keep the parameter in ctor.
Now we do the check if asked only once and store this into an internal attribute. Reason is that several methods require explicitly the check (`draw` for example).

Reply:
Ok for me